doc/release-notes-24118.md

```diff
@@ -0,0 +1,24 @@
+Add `sweepwallet` RPC
```

Comment:
b2b1d6b

- I think you can add your release note directly to the file `doc/release-notes.md` in this section:

```md
New RPCs
--------
```

- s/without unspecified/without a specified/
- add `(#24118)` to the end

<details><summary>suggested edit with fixups to work from</summary><p>

```text
The `sweepwallet` RPC spends the full balance of the provided UTXOs to one or
more receivers, without creating change.

By default, `sweepwallet` empties the wallet completely, leaving no UTXOs
behind. The optional `send_max` option allows skipping uneconomic UTXOs, thereby
maximizing the received amount.

The `sweepwallet` RPC can process a combination of receiver addresses, with and
without amounts specified. The specified amounts are paid first, and the
remainder is split among the receivers having unspecified amounts. At least one
address must be provided without a specified amount to receive the balance left
after fees.

Instead of the complete UTXO pool, the call can be used to create a transaction
from a specific UTXO set. We recommend using `sweepwallet` to empty wallets
or to spend specific UTXOs in full.

The `sweepwallet` RPC therefore provides a less cumbersome way of spending
specific UTXOs or emptying wallets than subtract fee from output/amount (SFFO).
If the user wishes to specify a budget rather than a set of UTXOs to delimit a
transaction, they should continue to use SFFO. (#24118)
```
</p></details>